packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -264,6 +269,8 @@ const drilldownSpawnAnimation = ref<'left' | 'right' | 'scale'>('scale');
 const route = useRoute();
 const router = useRouter();
 
+const warningBannerDismissed = ref(false);
+
```

Comment:
Can we just call this variable `warningBanner`?

Reply:
done